core/parsigex/parsigex.go

```diff
@@ -87,8 +88,11 @@ func (m *ParSigEx) handle(ctx context.Context, _ peer.ID, req proto.Message) (pr
 		return nil, false, errors.Wrap(err, "convert parsigex proto")
 	}
 
-	ctx, span := core.StartDutyTrace(ctx, duty, "core/parsigex.Handle")
-	defer span.End()
+	if duty.Type == core.DutyProposer || duty.Type == core.DutyBuilderProposer {
```

Comment:
I think we can safely assume `core.DutyBuilderProposer` isn't used anymore. I believe any version that uses it won't be compatible with the Charon version that will have this code, as it'll be before v1.

Reply:
Removed, thank you